added article 1

SriRaj910 opened the pull request:
Added an Article Titled "What is Lorem Ipsum?"

SriRaj910 committed:
added article 1

PVSASWIGGY reviewed (changes requested):
can you please add some more words?

SriRaj910 committed:
changes made
_
__________
1. Added What is lorem ipsum?
2.Added Why do we use it?
3. Added spaces to header and footer

PVSASWIGGY reviewed (approved):
matched requested word count.
